lib/main.coffee

```diff
@@ -10,7 +10,11 @@ module.exports =
     cursorPositionFormat:
       type: 'string'
       default: '%L:%C'
-      description: 'Format for the cursor position status bar element, where %L is the line number and %C is the column number'
+      description: 'Format for the cursor position indicator'
```

Comment:
I feel like this should include instructions to check the README for the supported replacements.